extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -467,16 +467,16 @@
 # See the file languages/SMW_LanguageXX.php to find what IDs to use for
 # datatpyes in your language. The default corresponds to "Type:Page".
 ##
-$GLOBALS['smwgPDefaultType'] = '_wpg';
+$smwDefaults['smwgPDefaultType'] = '_wpg';
 ##
 
 ###
 # Settings for OWL/RDF export
 ##
-$GLOBALS['smwgAllowRecursiveExport'] = false; // can normal users request recursive export?
-$GLOBALS['smwgExportBacklinks'] = true; // should backlinks be included by default?
+$smwDefaults['smwgAllowRecursiveExport'] = false; // can normal users request recursive export?
+$smwDefaults['smwgExportBacklinks'] = true; // should backlinks be included by default?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)